modules/ROOT/pages/index.adoc

```diff
@@ -121,4 +118,3 @@ xref:sap/sap-successfactors-connector.adoc[SAP SuccessFactors]
 == See Also
 
 * https://forums.mulesoft.com[MuleSoft Forum]
-* https://support.mulesoft.com/s/knowledge[Knowledge Base Articles]
```

Comment:
Why remove the KB articles?

Reply:
https://support.mulesoft.com/s/knowledge has been replace by https://help.mulesoft.com that covers much more than KB articles. I assume customers know about the Help Center.